views/editQuiz.handlebars

```diff
@@ -0,0 +1,36 @@
+<body>
+    {{> header}}
+
+    <div class="edit_quiz">
```

Comment:
Please reference and copy the quiz page to reduce the amount of nested `div`s and duplicated code. Everything that contains content should be a `content_container` and the grey div behind the cards should be a `full_width_contrast_container`.